account_partner_aged_statement_webkit/partner_aged_statement_data.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" ?>
+<openerp>
+    <!-- Mail template is done in a NOUPDATE block
+         so users can freely customize/delete them -->
+    <data noupdate="0">
+
+        <!--Email template -->
+        <record id="email_template_aged_statement" model="email.template">
+            <field name="name">Aged Statement Letter</field>
+            <field name="email_from">${(user.email or '')|safe}</field>
+            <field name="subject">${object.company_id.name} - Your aged balance</field>
+            <field name="email_recipients">${object.id or ''|safe}</field>
+            <field name="model_id" ref="account.model_res_partner"/>
+            <field name="auto_delete" eval="True"/>
+            <field name="report_template" ref="partner_aged_statement_report"/>
+            <field name="report_name">
+                ${(object.name or '').replace('/','_')}
+            </field>
+            <field name="body_html"><![CDATA[
+<div style="font-family: 'Lucica Grande', Ubuntu, Arial, Verdana, sans-serif; font-size: 12px; color: rgb(34, 34, 34); background-color: #FFF; ">
+
+    <!--<p>${user.company_id.overdue_msg.replace('\n', '&lt;br/&gt;')} </p>-->
+    % for line in user.company_id.overdue_msg.split('\n'):
+            <p> ${line}<br/> </p>
+    % endfor
+
+    <br/>
+    ${user.name}
+    <br/>
+    <br/>
+
+    <p style="border-left: 1px solid #8e0000; margin-left: 30px;">
+       &nbsp;&nbsp;<strong>REFERENCES</strong><br />
```

Comment:
Instead of translating this whole html content, you could instead use the translating function to translate some fields only. `${_("REFERENCES")}` and `${_("Total overdue amount: ")}`